webtester-core/src/test/java/integration/pageobjects/SingleSelectIntegrationTest.java

```diff
@@ -0,0 +1,187 @@
+package integration.pageobjects;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.NoSuchElementException;
+
+import integration.AbstractWebTesterIntegrationTest;
+
+import info.novatec.testit.webtester.api.annotations.IdentifyUsing;
+import info.novatec.testit.webtester.api.exceptions.WrongElementClassException;
+import info.novatec.testit.webtester.pageobjects.PageObject;
+import info.novatec.testit.webtester.pageobjects.SingleSelect;
+
+
+
+public class SingleSelectIntegrationTest extends AbstractWebTesterIntegrationTest {
```

Comment:
Can not change assertions for `testThatSelectedTextIsEmpty_emptySelect()` (line 108), and `testThatSelectedValueIsEmpty_emptySelect()` (line 128) 
from  `assertThat(someString.isEmpty(), is(true));` to `assertThat(someString, is(empty()));`

And can not change assertion for `testThatIndexIsNull_emptySelect()` (line 148)
from `assertThat(index.equals(null), is(true));` to `assertThat(index, is(null));`, because null is ambigious.

Do you know why? Is it because `someString` is a String and not a List?